src/schema/objects/enums.yaml

```diff
@@ -360,14 +360,19 @@ individual:
     In context of surfaces this space has been referred to as `fsnative`.
 
     In order for this space to be interpretable, `SpatialReference` metadata MUST be provided.
-study:
+study__space:
   value: study
   display_name: study
   description: |
     Custom space defined using a group/study-specific template.
     This coordinate system requires specifying an additional file to be fully defined.
 
     In order for this space to be interpretable, `SpatialReference` metadata MUST be provided.
+study__datasettype:
+  value: study
+  display_name: study
+  description: |
+    A study BIDS dataset.
```

Comment:
```suggestion
    A study BIDS dataset to organize the data for the entire study (original source data, raw BIDS, derivatives) as a valid BIDS dataset.
```

Reply:
eh, greedy me saw this as a potential for "common denominator" as e.g. even an empty BIDS dataset before any subject data is collect could be a valid "study" one and then re-assigned to "raw".  But you are right, we better make it cleaner and following the example in the text aim to cover the entire study (only) in this description instead of staying vague. Let's do it, thanks